lms/urls.py

```diff
@@ -685,6 +685,16 @@
     ),
 ]
 
+
+urlpatterns += [
+    url(
+        r'^bundles/{}/about'.format(
+            r'(?P<bundle_uuid>[0-9a-f-]+)',
+        ),
+        include('openedx.features.journals.urls')
+    ),
+]
```

Comment:
Use Django App Plugin: DAP will allow you to declare your own urls without needing to add them into the platform.

Reply:
fixed